src/reducers/note.js

```diff
@@ -1,24 +1,29 @@
 import { EditorState } from 'draft-js';
+import getNotes from '../lib/noteServices';
 
 const initState = {
-  notes: [
-    { id: 0, title: 'note 0', text: 'this is text for note 0' },
-    { id: 1, title: 'note 1', text: 'this is text for note 1' },
-    { id: 2, title: 'note 2', text: 'this is text for note 2' },
-  ],
+  notes: [],
   currentNote: { id: 2, title: 'note 2', text: 'this is text for note 2' },
   editorState: EditorState.createEmpty(),
 };
 
 const NOTE_ADD = 'NOTE_ADD';
 const EDITOR_UPDATE = 'EDITOR_UPDATE';
+const NOTES_LOAD = 'NOTES_LOAD';
```

Comment:
@laurelherting EDITOR_UPDATE is still being used. `NOTES_LOAD` is a constant so you don't run the risk of mistyping a string. Better to use a constant.